packages/sdk/src/types/network/payloads/assets.ts

```diff
@@ -9,6 +9,13 @@ import { AssetLike, AssetSource } from '../../runtime/assets';
 
 export type CreateColliderType = ColliderType | 'none';
 
+/** @hidden */
+export type LoadSound = Payload & {

```

Comment:
>LoadSound [](start = 12, length = 9)

I don't think this message is necessary. See related comments in assetManager.ts

Reply:
Agreed. Use the CreateAsset payload below.

---
In reply to: [257468904](https://github.com/Microsoft/mixed-reality-extension-sdk/pull/204#discussion_r257468904) [](ancestors = 257468904)

Reply:
I'll look at this. At the same time I will try to see if we can come up with a better name - I really don't like the "Create" term for loaded assets - that just feels weird.